gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -1,5 +1,71 @@
 """Script to create release sites HT for v4 genomes."""
 import hail as hl
+from gnomad.utils.vep import filter_vep_transcript_csqs
+
+# TODO: steps to create release sites HT
+# input: VRS-annotated sites HT v3.1.4
+# move SIFT and PolyPhen to insilico_predictors in vep_ht
+# remove missing fields in vep_ht
+# drop vep from site HT and merge with vep_ht
+# remove old insilico predictors and replace with new ones (the left predictors)
+# update dbSNP to dbSNP 156
+# update freq, freq_meta, freq_index_dict by integrating updates in HGDP/TGP
+# rerun inbreeding_coefficient with the callstats from the new freq
+# replace oth with remaining in global fields
+# update global fields
+
+
+def get_sift_polyphen_from_vep(ht: hl.Table) -> hl.Table:
+    """
+    Get SIFT and PolyPhen scores from VEP 105 annotations.
+
+    .. note::
+     This function is to get a max of SIFT and PolyPhen scores from mane_select
+     transcripts, otherwise get a max of SIFT and PolyPhen scores from canonical
+     transcripts. It will also drop the SIFT and PolyPhen scores and predictions
+     from the transcript_consequences struct.
+
+    :param ht: VEP105 annotated Hail Table.
+    :return: Hail Table with VEP105 annotations and  SIFT and PolyPhen scores
+    extracted and stored in insilico_predictors struct.
+    """
+    mane = filter_vep_transcript_csqs(
+        ht, synonymous=False, canonical=False, mane_select=True
+    )
+    canonical = filter_vep_transcript_csqs(ht, synonymous=False, canonical=True)
+    ht = ht.annotate(
+        sift_mane=mane[ht.key].vep.transcript_consequences.sift_score,
+        polyphen_mane=mane[ht.key].vep.transcript_consequences.polyphen_score,
+        sift_canonical=canonical[ht.key].vep.transcript_consequences.sift_score,
+        polyphen_canonical=canonical[ht.key].vep.transcript_consequences.polyphen_score,
+    )
+
+    ht = ht.annotate(
+        vep=ht.vep.annotate(
+            transcript_consequences=ht.vep.transcript_consequences.map(
+                lambda x: x.drop(
+                    "sift_prediction",
+                    "sift_score",
+                    "polyphen_prediction",
+                    "polyphen_score",
+                )
+            )
+        )
+    )
+    ht = ht.annotate(
+        insilico_predictors=hl.struct(
+            sift_max=hl.or_else(hl.max(ht.sift_mane), hl.max(ht.sift_canonical)),
+            polyphen_max=hl.or_else(
+                hl.max(ht.polyphen_mane), hl.max(ht.polyphen_canonical)
+            ),
+        )
+    )
```

Comment:
It would be more readable if the insilico_predictors struct was built in a single step rather than starting it here. For that reason, I think the returned ht could just be a 
```
ht = ht.select(
    sift_max=hl.or_else(hl.max(ht.sift_mane), hl.max(ht.sift_canonical)),
    polyphen_max=hl.or_else(hl.max(ht.polyphen_mane), hl.max(ht.polyphen_canonical)
)
```